build/charts/antrea/crds/ippool.yaml

```diff
@@ -9,7 +9,7 @@ spec:
   versions:
     - name: v1alpha2
       served: true
-      storage: true
+      storage: false
```

Comment:
Could you add the deprecated field for this old version. We plan to move the minimal supported version of K8s to v1.19: https://github.com/antrea-io/antrea/issues/5879.

Reply:
OK, got it.